dynamiq/connections/connections.py

```diff
@@ -1232,3 +1233,62 @@ def conn_params(self) -> dict:
             "api_base": self.url,
             "api_key": self.api_key,
         }
+
+
+class MPC(BaseConnection):
+    # SSE
```

Comment:
Let's add proper Pydantic model validator to validate mandatory params for each of connection types. Let's do this on connection init, not on client init